src/v3/src/transformer/layout/securityQuestion/securityQuestionEnroll.ts

```diff
@@ -120,43 +124,61 @@ export const transformSecurityQuestionEnroll: IdxStepTransformer = ({ transactio
     },
   };
 
+  const customOptions: StepperRadioElement['options']['customOptions'] = [{
+    value: 'predefined',
+    label: loc('oie.security.question.questionKey.label', 'login'),
+    callback: (widgetContext, stepIndex) => {
+      const { dataSchemaRef, setData } = widgetContext;
+      dataSchemaRef.current!.submit = predefinedSubmitButton.options;
+      dataSchemaRef.current!.fieldsToValidate = ['credentials.answer'];
+
+      // reset default selected question
+      setData((prev) => ({
+        ...prev,
+        [QUESTION_KEY_INPUT_NAME]: preSelectedQuestion,
+      }));
+
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
Oof, looks like these dependencies are pretty tangled. No easy way to define things ahead of time to avoid this I'm guessing?

Reply:
Actually I was able to figure out a way, will push up with my next commit.